logic/Rule.java

```diff
@@ -122,9 +123,6 @@ public Conclusion conclusion() {
         return conclusion;
     }
 
-    public Map<Identifier, Concept> putConclusion(ConceptMap whenConcepts, TraversalEngine traversalEng, ConceptManager conceptMgr) {
-        return conclusion.putConclusion(whenConcepts, traversalEng, conceptMgr);
-    }
```

Comment:
remove `Rule.putConclusion()`, as `Rule.conclusion().materialise()` is more idiomatic for us